rama-cli/src/cmd/tls/mod.rs

```diff
@@ -0,0 +1,158 @@
+use clap::Args;
+use rama::{
+    Context, Layer, Service,
+    error::BoxError,
+    net::{
+        address::Authority,
+        client::{ConnectorService, EstablishedClientConnection},
+        tls::{
+            DataEncoding,
+            client::{ClientConfig, NegotiatedTlsParameters},
+        },
+    },
+    tcp::client::{Request, service::TcpConnector},
+    tls::{
+        rustls::client::{TlsConnectorData, TlsConnectorLayer},
+        std::dep::boring::x509::X509,
+    },
+};
+use tokio::net::TcpStream;
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::{EnvFilter, fmt, layer::SubscriberExt, util::SubscriberInitExt};
+
+#[derive(Args, Debug, Clone)]
+/// rama tls support
+pub struct CliCommandTls {
+    /// The address to connect to
+    /// e.g. "example.com" or "example.com:8443"
+    /// if no port is provided, the default port 443 will be used
+    address: String,
+
+    #[arg(long, short = 'k')]
+    /// Wether to skip certificate verification
+    insecure: bool,
```

Comment:
this value is ignored for now it seems, you'll want to use it for the `server_verify_mode` in https://docs.rs/rama/latest/rama/net/tls/client/struct.ClientConfig.html

Reply:
Yes forgot to add it, plz check last commit